src/main/java/com/qaprosoft/carina/demo/phpTravels/components/Table.java

```diff
@@ -41,6 +41,12 @@ public void deleteBooking(String id) {
             }
         }
     }
-
-
+    public boolean getBookingIdExist(String id) {
```

Comment:
get is used when we return some text, number, count etc
when we verify that something is shown we use isSomethingShown()
IN your case it'll be: `isBookingShown(String id)`